tests/virt/upgrade_custom/vgpu/test_vgpu_vm_upgrade.py

```
@@ -0,0 +1,54 @@
+]
+
+
+class TestUpgradeVGPU:
+    """Pre-upgrade test"""
+
+    @pytest.mark.polarion("CNV-11782")
+    @pytest.mark.order(before=IUO_UPGRADE_TEST_ORDERING_NODE_ID)
+    @pytest.mark.dependency(name=f"{VIRT_NODE_ID_PREFIX}::test_vgpu_vm_before_upgrade")
+    def test_vgpu_vm_before_upgrade(self, supported_gpu_device, rhel_vm_for_upgrade_session_scope):
+        running_vm(vm=rhel_vm_for_upgrade_session_scope, wait_for_cloud_init=True)
+        verify_gpu_device_exists_in_vm(vm=rhel_vm_for_upgrade_session_scope, supported_gpu_device=supported_gpu_device)
```

Pylint (3.3.7)

[convention] Line too long (119/100)

(C0301)